roles/contiv_network/tasks/main.yml

```diff
@@ -68,3 +68,6 @@
   args:
     chdir: /usr/bin/
     creates: contivctl
+
+- include: aci_tasks.yml
+  when: (run_as == "master") and (aci_mode is defined) and (aci_mode|bool)
```

Comment:
I did think of the default mode for aci_mode, but was not sure if it should be in group-vars/all or under contiv_network role. Was waiting for your input. :) And I'll make it contiv-network-mode to take in "standalone/aci" mode for now.